### ISSUE TITLE
Question: How to use ReorderPlaylist

### ISSUE THREAD

**Issue by KenthJ** [JohnnyCrazy/SpotifyAPI-NET#534]
Hi there

Thanks a lot for all your effort to make this API

I can't figure out how to use the reorder method, like below:
spotify.Playlists.ReorderItems("myPlayListID", PlaylistReorderItemsRequest)

How do I create this PlaylistReorderItemsRequest?

A couple of code lines would be really nice.

It seems like the documentation is not up to date here, or at least I aven't been able to find it.

Any help is greatly appreciated.

**Comment by KenthJ** [comment]
Hmm, have tried some more, and found a solution which can compile without errors, but it still doesn't work

I'm using the code below, but an error happens during the execution of the reorder code line, only Trace1 is written to my output, but no errors are written to my output, and it never writes out Trace2, any good ideas

public async Task Reorder(string playlistId)
        {
            System.Diagnostics.Debug.WriteLine("Trace1");
            var response = await mySpotify.Playlists.ReorderItems(playlistId, new PlaylistReorderItemsRequest(3, 1));
            System.Diagnostics.Debug.WriteLine("Trace2");
        }

**Comment by KenthJ** [comment]
I tried to update to latest NuGet version 6.0.0 released yesterday

But it still fails, I get following info in the output window:

Trace1
Exception thrown: 'SpotifyAPI.Web.APIException' in mscorlib.dll

**Comment by JohnnyCrazy** [comment]
Hi @KenthJ 

Make sure to use the inline XML Docs, your editor should show you the hints when hovering over the class. If not, you can also look at the source files: https://github.com/JohnnyCrazy/SpotifyAPI-NET/blob/master/SpotifyAPI.Web/Models/Request/PlaylistReorderItemsRequest.cs

The exception is important here, what does the message say?

`new PlaylistReorderItemsRequest(3, 1)` means move the 3rd item in the playlist after the 1st item in the playlist.

**Comment by KenthJ** [comment]
Thanks a lot for your response, and it actually led me in the direction, so I was able to fix the problem.

For some reason the error message was not written to the output window, but with a try catch, I was able to catch the error message.

The error message was "Insufficient client scope"

So I added the Scopes.PlaylistModifyPrivate to the scope, and then it worked like a charm.

Thanks you so much for your help.
